pkgs/tools/security/hashdeep/default.nix

```diff
@@ -11,6 +11,13 @@ stdenv.mkDerivation rec {
     sha256 = "0m2b042ndikavmplv3qjdhfj44hl1h8car83c192xi9nv5ahi7mf";
   };
 
+  patches = [
+    (fetchpatch {
+        url = "https://github.com/jessek/hashdeep/commit/6ef69a26126ee4e69a25392fd456b8a66c51dffd.patch";
```

Comment:
```suggestion
        name = "fix-cpp-defect-report-1512.patch";
        url = "https://github.com/jessek/hashdeep/commit/6ef69a26126ee4e69a25392fd456b8a66c51dffd.patch";
```

Otherwise we end up with loads of anonymous patches floating around.

Reply:
Good remark, thank you!